### PR TITLE
feat: network error handling forgot password

### PR DESCRIPTION
**Pull request by sudoDeznit**

### CODE REVIEW

**Comment by sudoDeznit** [comment]
@xlcrr this is completed PR for now

File: actions/auth_actions.js [hunk 1]
```diff
@@ -229,56 +229,57 @@ export const changeServerStatusText = text => {
...
-                        success: false,
-                        msg: error.response.data.errors.email // We can't find a user with that email address
-                    };
+        let response;
+        try {
+            response = await axios(URL + '/api/password/email', {
+                method: 'POST',
+                data: {
+                    email: email,
+                    api: true // we need this to override the response
```
Comment (xlcrr) [inline actions/auth_actions.js:244]: Is this still needed? Seems irrelevant and looks like an early way of fixing something quickly

**Review by xlcrr — APPROVED** [review]

**Comment by xlcrr** [comment]
See 1 comment @sudoDeznit

File: actions/auth_actions.js (hunk 1 above)
Comment (sudoDeznit) [reply actions/auth_actions.js:244]: Will check this in remove if not needed.. will add it in future PR..
Made a trello card to track this
[trello card](https://trello.com/c/3sgbZfB0)
